disable_warnings.sh

```diff
@@ -0,0 +1,38 @@
+#floriantschopp/schneith, 07/2019
+#!/bin/bash
+
+# Check for arguments
+if [ -z "$1" ]; then
+  echo "Disable the warnings generated by the eigen core headers by adding pragma's to all header files.";
+  echo -e "Usage: disable_warnings.sh CATKIN_DEVEL_SPACE/EIGEN_SYSTEM_INCLUDE"
+  exit -1
+fi
+
+DEVEL_SPACE=$1
+EIGEN_INCLUDES_DIR=$1/include/eigen3
+CURR_DIR=$(pwd -P)
+TEMPLATE_DIR=$(pwd -P)/template
+cd $EIGEN_INCLUDES_DIR
+CORE_DIR=$(pwd -P)/Eigen/src/Core
+cd $CURR_DIR
```

Comment:
There is a more concise way without changing the directory in the current `bash`: Change the directory only in the sub shell used for the expansion of `$(...)`:

```sh
CORE_DIR=$(cd $EIGEN_INCLUDES_DIR; pwd -P)/Eigen/src/Core
```
This would replace 4 of the above 5 lines (except the `TEMPLATE_DIR=$(pwd -P)/template`).